src/examples/can/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"device/sam"
+	"fmt"
+	"machine"
+	"time"
+)
+
+func main() {
+	can1 := machine.CAN{Bus: sam.CAN1}
```

Comment:
I think it would be better if the machine package defined `CAN1` this way, so you can use `machine.CAN1` directly. For example, like this:

```go
var CAN1 = &CAN{Bus: sam.CAN1}
```

Reply:
Indeed.
Changed.